### ISSUE TITLE
V1 version cannot be train[Task]: 

### ISSUE THREAD

**Issue by MMYY-yy** [openvinotoolkit/anomalib#1684]
### What is the motivation for this task?

error: Parser key "post_processing.normalization_method":
  Expected a member of <enum 'NormalizationMethod'>: {CDF,MIN_MAX,NONE}. Got value: min_max

### Describe the solution you'd like

error: Parser key "post_processing.normalization_method":
  Expected a member of <enum 'NormalizationMethod'>: {CDF,MIN_MAX,NONE}. Got value: min_max
  
  class_path: anomalib.data.Folder
init_args:
  root: "data/SY1"
  normal_dir: "SY1_OK_train"
  abnormal_dir: "SY1_NG_test"
  normal_test_dir: "SY1_OK_test"
  mask_dir: null #"ground_truth/broken_large"
  normal_split_ratio: 0
  extensions: null #[".png"]
  image_size: [320, 95]
  center_crop: null
  normalization: imagenet
  train_batch_size: 32
  eval_batch_size: 32
  num_workers: 8
  task: CLASSIFICION #SEGMENTATION
  transform_config_train: null
  transform_config_eval: null
  test_split_mode: FROM_DIR
  test_split_ratio: 0.2
  val_split_mode: SAME_AS_TEST
  val_split_ratio: 0.5
  seed: null

model:
  class_path: anomalib.models.Cfa
  init_args:
    backbone: wide_resnet50_2
    gamma_c: 1
    gamma_d: 1
    num_nearest_neighbors: 3
    num_hard_negative_features: 3
    radius: 1.0e-05
    

metrics:
        #pixel: AUROC
  image: AUROC
trainer:
  max_epochs: 30
  callbacks:
    - class_path: lightning.pytorch.callbacks.EarlyStopping
      init_args:
        patience: 5
        monitor: image_AUROC #pixel_AUROC
        mode: max

**Comment by ashwinvaidya17** [comment]
I am closing this issue. Feel free to re-open it if it still persists.

**Comment by MMYY-yy** [comment]
Thank you very much for your reply. I will continue to verify the algorithm

---- Replied Message ----
| From | Ashwin ***@***.***> |
| Date | 02/14/2024 23:09 |
| To | openvinotoolkit/anomalib ***@***.***> |
| Cc | MMYY-yy ***@***.***>,
Author ***@***.***> |
| Subject | Re: [openvinotoolkit/anomalib] V1 version cannot be train[Task]: (Issue #1684) |

I am closing this issue. Feel free to re-open it if it still persists.
